pkg/provider/acme/provider.go

```diff
@@ -80,6 +80,8 @@ type DNSChallenge struct {
 	DelayBeforeCheck        ptypes.Duration `description:"Assume DNS propagates after a delay in seconds rather than finding and querying nameservers." json:"delayBeforeCheck,omitempty" toml:"delayBeforeCheck,omitempty" yaml:"delayBeforeCheck,omitempty" export:"true"`
 	Resolvers               []string        `description:"Use following DNS servers to resolve the FQDN authority." json:"resolvers,omitempty" toml:"resolvers,omitempty" yaml:"resolvers,omitempty"`
 	DisablePropagationCheck bool            `description:"Disable the DNS propagation checks before notifying ACME that the DNS challenge is ready. [not recommended]" json:"disablePropagationCheck,omitempty" toml:"disablePropagationCheck,omitempty" yaml:"disablePropagationCheck,omitempty" export:"true"`
+	PropagationRNS          bool            `description:"Use all the recursive nameservers to check the propagation of the TXT record." json:"propagationRNS,omitempty" toml:"propagationRNS,omitempty" yaml:"propagationRNS,omitempty" export:"true"`
```

Comment:
```suggestion
	PropagationRNS          bool            `description:"Use all the recursive name servers to check the propagation of the TXT record." json:"propagationRNS,omitempty" toml:"propagationRNS,omitempty" yaml:"propagationRNS,omitempty" export:"true"`
```

Reply:
For the DNS word, "nameservers" is used as one word: https://developers.cloudflare.com/dns/nameservers/